app/helpers/blocks_helper.rb

```diff
@@ -0,0 +1,24 @@
+module BlocksHelper
+  def block_item(block)
+    content_tag(:td, block.title)
+
+    if block.current?
+      content_tag(:td) do
+        link_to 'Сделать обычной', reset_as_current_block_path(block), method: :put
+      end
+    else
+      content_tag(:td) do
+        link_to 'Сделать текущей', set_as_current_block_path(block), method: :put
+      end
+    end
+
+    content_tag(:td) do
+      link_to 'Редактировать', edit_block_path(block)
+    end
+
+    content_tag(:td) do
+      link_to 'Удалить', block, method: :delete,
+                    data: { confirm: 'Удалить колоду?' }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.